src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/ResetTask_2.sql

```diff
@@ -58,19 +55,8 @@ ELSE
                    RetryCount        = @retryCount + 1
```

Comment:
Reset set task status to 1, like a new queued task, so there is not a worker until someone pick it up by calling GetNextTask, and the worker column will be set in that SP. 
I believe leave the old worker here have no harm and can provide us who reset this task.

Reply:
Resetting to status = 1 make task available to be picked up. Most likely this will change the worker. Logic to puck items with status = 1 does currently have worker/date preservation logic. So either that needs to be added to get procedure or changed here.

Reply:
Pick up logic is in GetNextTask, there is already the login about adding the worker

Reply:
That logic preserves Worker and StartDate only for timed out tasks. Right?

Reply:
As we discussed, add restartinfo containing worker and startdatetime here.